### PR TITLE
chore: log parsing pipelines happy path integration test

### PR DESCRIPTION
**Pull request by raj-k-singh**
Contributes towards  https://github.com/SigNoz/signoz/issues/1779

### CODE REVIEW

**Comment by srikanthccv** [comment]
I haven't had a chance to work/review/collaborate with these parts of code for quite a long. I will take some time to provide any meaningful review. But feel free to merge if you Nitya approves it.

File: pkg/query-service/tests/integration/logparsingpipeline_test.go [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+	)
+	if err != nil {
+		tb.t.Fatalf("couldn't create authenticated test request: %v", err)
+	}
+
+	respWriter := httptest.NewRecorder()
+	tb.apiHandler.CreateLogsPipeline(respWriter, req)
+
+	response := respWriter.Result()
+	responseBody, err := ioutil.ReadAll(response.Body)
```
Comment (nityanandagohain) [inline pkg/query-service/tests/integration/logparsingpipeline_test.go:227]: ioutil.ReadAll is depircated

**Review by nityanandagohain — COMMENTED** [review]
Overall looks good to me, please look at the comments.

Also make sure to to include these tests in makefile so that we can run `make tests`

File: pkg/query-service/tests/integration/logparsingpipeline_test.go [hunk 2]
```diff
@@ -0,0 +1,542 @@
...
+				Name:    "pipeline1",
+				Alias:   "pipeline1",
+				Enabled: true,
+				Filter:  "attributes.method == \"GET\"",
+				Config: []model.PipelineOperator{
+					{
+						OrderId: 1,
+						ID:      "add",
+						Type:    "add",
+						Field:   "body.test",
```
[the remark this answers is not in the document]
Comment (nityanandagohain) [reply pkg/query-service/tests/integration/logparsingpipeline_test.go]: so body can be used in some other operators live move, so lets leave it.

**Review by nityanandagohain — APPROVED** [review]
It's good to merge, but I will have a look again if something needs to be added/changed.

**Review by srikanthccv — APPROVED** [review]
One approval should be enough. Approving to get this merged.

**Pull request merged**